Source/WebCore/platform/graphics/Damage.h

```diff
@@ -44,7 +44,15 @@ class Damage {
         Unified,
     };
 
-    Damage() = default;
+    static constexpr int s_tileSize { 256 };
+
+    Damage()
+        : m_tileSize(s_tileSize, s_tileSize)
+    {
+        // FIXME: Add a constructor to pass the size.
+        resize({ 2048, 1024 });
```

Comment:
shouldn't we have 2048x1024 as a `constexpr`, too? Or you're expecting this is a very short-lived FIXME?

Reply:
Yes, this is temporary.